Docs/Configuration.tex

```diff
@@ -340,11 +349,12 @@ \subsection{Configuration Processing}\label{configuration-processing}
 
 \subsection{Configuration Structure}\label{configuration-structure}
 
-\texttt{OC\ config} is separated into following sections, which are described
-in separate sections of this document. By default it is tried to not enable
-anything and optionally provide kill switches with \texttt{Enable} property
-for \texttt{plist dict} entries. In general the configuration is written
-idiomatically to group similar actions in subsections:
+The \texttt{OC\ config} file is separated into subsections, as described in
+separate sections of this document, and is structured so as to attempt not to
```

Comment:
It is not structured to not enable anything, but rather designed or similar. Structuring is implying the form not the meaning.